bigtop-packages/src/common/logstash/do-component-build

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+. `dirname $0`/bigtop.bom
+
+export RELEASE=1
+export LANG="en_US.UTF-8"
+
+rake bootstrap
```

Comment:
https://ci.bigtop.apache.org/view/Test/job/Build-Deploy-Smoke-Test-Pull-Request-All-Distros/OS=centos-7-x86_64-deploy/53/consoleFull

```
+ rake bootstrap
/bigtop/build/logstash/rpm/SOURCES/do-component-build: line 24: rake: command not found
error: Bad exit status from /var/tmp/rpm-tmp.LphHXs (%build)
    Bad exit status from /var/tmp/rpm-tmp.LphHXs (%build)

FAILURE: Build failed with an exception.

* Where:
Script '/bigtop/packages.gradle' line: 500

* What went wrong:
Execution failed for task ':logstash-rpm'.
> Process 'command 'rpmbuild'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 24s
```

Reply:
The rake would be installed when ruby was installed in bigtop toolchain.
`include bigtop_toolchain::ruby`

It seems we should rebuild the provisioner docker image to add ruby installation.

Reply:
Yeah. I think you mean the bigtop/slaves images, right?